tests/snapshots/src/main/generated/ujson/IndexedValue.scala

```diff
@@ -34,6 +34,14 @@ sealed trait IndexedValue {
   def index: Int
 //    ^^^^^ definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/IndexedValue#index().
 //          documentation ```scala\ndef index: Int\n```
+//          relationship is_reference is_implementation ujson/IndexedValue.Arr#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.False#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.Null#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.Num#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.NumRaw#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.Obj#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.Str#index.
+//          relationship is_reference is_implementation ujson/IndexedValue.True#index.
```

Comment:
Implementation-wise I think the PR is fine, I'm just concerned about plugging the holes in the backend with something that increases the size of the SCIP index.

This example is probably on the extreme end of the spectrum

Reply:
I'm not so concerned about index size getting bloated from this change. These relationships represent a tiny part of the payload since most of the overhead comes from basic occurrences (just a single `import` statement can emit 5-6 occurrences). Symbols also have a lot of redundancy that gets compressed away.